src/app.py

```
@@ -72,4 +72,4 @@ def update_user():
     try:
         app.run(host="0.0.0.0", debug=True)
     except NoCredentialsError:
-        print("Error: AWS credentials not found. Make sure to set up your credentials.")
+        print("Error: AWS credentials not found. Make sure to set up your credentials.")
```

raise exception as above instead of printing.